tensorflow/python/keras/layers/gru_v2_test.py

```diff
@@ -140,6 +141,9 @@ def test_from_config_GRU(self):
 
   @test_util.run_v2_only
   def test_gru_v2_feature_parity_with_canonical_gru(self):
+    if build_info.is_rocm_build:
+      self.skipTest("Skipping the test as ROCm MIOpen does not support padded input yet.")
```

Comment:
Please split the error message into 2 lines, which fit into 80 char limit. Please apply this to all the tests.

Reply:
Done